x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/PlannerUtils.java

```diff
@@ -238,11 +237,11 @@ public static ElementType toElementType(DataType dataType) {
         if (dataType == EsQueryExec.DOC_DATA_TYPE) {
             return ElementType.DOC;
         }
-        // TODO: Spatial types can be read from source into BYTES_REF, or read from doc-values into LONG
-        if (dataType == EsqlDataTypes.GEO_POINT) {
-            return ElementType.BYTES_REF;
-        }
-        if (dataType == EsqlDataTypes.CARTESIAN_POINT) {
+        // TODO: Spatial types can be read from source into BYTES_REF, or read from doc-values
```

Comment:
Perhaps this comment can be removed, as it was a placeholder for me to re-visit this code with the doc-values work, but that PR took another direction.

Reply:
```suggestion
```